recipe/meta.yaml

```diff
@@ -67,6 +62,8 @@ requirements:
     - scipy >=1.0
     # CUDA Python 11.6 or later
     - cuda-python >=11.6
```

Comment:
I guess that because numba is a compiler, you don't have to compile GPU and CPU versions of numba? One version compiles for both CPU and GPU?

Reply:
These are run_constrained. I assume if cuda is there it will make use of it.